### DISCUSSION TITLE
Qtable not rendering on subsequent changes when rows data updates.throws recursive vue warning

### DISCUSSION THREAD

**Discussion by jballa1** [quasarframework/quasar#14583] [General - Components / Directives / etc]
### What happened?

I am trying to fetch data from API and update the table as query changes. However qtable update only ones and there is a vue warning saying that there is a recursive loop updated maximum number of allowed loops. On subsequent changes in the rows data, the table wont update

### What did you expect to happen?

Table should update for every data change in the rows data

### Reproduction URL

https://stackblitz.com/edit/quasarframework-w32tum

### How to reproduce?

1. create any table
2. fetch data from api. .. may be you can try example api that I have used (https://sarvail.net/wp-json/ds-custom_endpoints/v1/posts?page=1&per_page=10&category=alumni-news. 
3. fetch the data. see how the vue warns about recursive
4. change any query parameters like page, perpage ...you get data correctly but the table wont update

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Safari, Microsoft Edge, iOS

### Quasar info output

```Shell
runtime-core.esm-bundler.js:38 
        
       [Vue warn]: Invalid VNode type: undefined (undefined) 
  at <QTable title="Treats" rows= (10) [{…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}] loading-label="DATA IS LOADING" > 
  at <DashboardPage onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy {__v_skip: true} > > 
  at <RouterView> 
  at <QPageContainer> 
  at <QLayout view="hHh lpR fFf" class="bg-grey-1" > 
  at <MainLayout onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy {__v_skip: true} > > 
  at <RouterView> 
  at <App>
```

**Comment by jballa1** [comment]
In steps, dont add anything to columns. Let it select all the columns

**Comment by pdanpdan** [comment]
Make a repro that works, please.

**Reply by jballa1** [reply]
Please see the project link below. After application loads you will see two button that fetch two different results. The table updates for first click but when you click the other button, the table is not getting updated.

Note: if you add only specific column it works fine but I dont want specify columns manually and let it drive from the fields supplied by back end

https://stackblitz.com/edit/quasarframework-ot6mm5

**Comment by pdanpdan** [comment]
It's just because you break the table by making it display objects, arrays, ...
https://stackblitz.com/edit/quasarframework-95xryj?file=src%2Flayouts%2FMainLayout.vue

**Reply by jballa1** [reply]
Thanks for the reply. If I understand your comment, the data that we are fetching from the url contains objects ,Array ..ex 

Thank you.
